packages/x-components/src/x-modules/next-queries/components/next-queries-list.vue

```diff
@@ -136,6 +151,14 @@
       if (this.nextQueriesAreOutdated) {
         return this.injectedListItems;
       }
+      if (
+        this.concatWhenNoMoreItems &&
+        !this.hasMoreItems &&
+        this.injectedListItems.length &&
+        this.offset > this.injectedListItems.length
```

Comment:
Can you refactor this out and into a computed property, like we're doing with `nextQueriesAreOutdated`?